frontend/src/qri/queryParams.ts

```diff
@@ -0,0 +1,6 @@
+export interface QueryParams {
+  q?: string
+  sort?: 'name' | 'recentlyupdated'
+  page?: number
+  pagesize?: number
+}
```

Comment:
move this into `src/qri/search.ts` and rename to `SearchParams`. I totally get the tension of passing a specific thing to the `PageControl` component, we can extract a generalized interface that serves PageControl when the need arises down the road.

Add a constructor function to create `SearchParams`:

```tsx
export function NewSearchParams(d: Record<string,any>): SearchParams {
  return {
    q: d.q || '',
    sort: d.sort || 'recentlyupdated',
    page: d.page,
    pagesize: d.pagesize,
  }
}
```

Import & use this constructor whenever you're allocating `SearchParams`, like in `features/search/Search.tsx`